src/js/modules/rpc/service.ts

```diff
@@ -24,25 +23,6 @@ const readConfigFile = (confPath: string): Promise<Record<string, any>> => {
   }
 };
 
-/*
```

Comment:
I would move this commit to the beginning of the PR and rename it to be `Removing wasm engines use of the filesystem to store/manage scripts`